src/components/Models/Model/Model.component.tsx

```diff
@@ -29,6 +50,28 @@ export const Model: FC<ModelProps> = ({ scene, scale = 1, modelRef, onLoaded, on
 
   useEffect(() => triggerCallback(onLoaded), [scene, materials, onLoaded]);
 
+  useEffect(() => {
+    gl.domElement.addEventListener('mousedown', setTouchingOn);
+    gl.domElement.addEventListener('touchstart', setTouchingOn);
```

Comment:
as we discover touch events on phone looks unnatural we need to check what's going on there